lib/document-to-html.js

```diff
@@ -224,12 +224,21 @@ function DocumentConversion(options, comments) {
         return idPrefix + suffix;
     }
 
-    var defaultTablePath = htmlPaths.elements([
```

Comment:
This change is a larger extension of the original implementation and now the old behaviour is quite hidden (getDefaultTable(element)=defaultTablePath, if element has no width or a width without dxa unit.). This makes it harder to rebase the fork and see later what we actually patched/changed. I suggest to make the patches more obvious via:

1. An extension nodejs module for document-to-html
2. Create an extension that takes the existing defaultTablePath implementation additionally as input.
3. Use the extension in the document-to-html.js file, i.e. instead of
    `return findHtmlPath(element, defaultTablePath).wrap(function()`
   use
   `return findHtmlPath(element, extension.defaultTablePath(element, defaultTablePath)`
   
It then communicates quite explicitely: a) we are patching defaultTablePath and b) our patch is based on the existing defaultTablePath and element props in addition.

Reply:
It seemed easier to change the creation of the `deafultTablePath` as you pass the style attribute at that point. Perhaps it's possible to modify it once it's already created but as it's only used in this one case anyway it didn't seem like a big change. I'll see what I can do but as far as I can tell, mammoth is pretty much dead when it comes to adding new features and I don't think we'll ever want to merge this back into the main fork